helm-chart/values.yaml

```diff
@@ -24,7 +24,7 @@ cvat:
       additionalVolumeMounts: []
     worker:
       export:
-        replicas: 2
+        replicas: 1
```

Comment:
Why did you change the default value? From my point of view, it is better to have a similar number of workers for k8s and docker-compose

Reply:
We can drop a comment why do we have 2 replicas by default maybe? Just for people to know what it is and why they should or shouldn't change it